### ISSUE TITLE
Login issues when self-signed-cert under specific circumstances

### ISSUE THREAD

**Issue by k9ert** [cryptoadvance/specter-desktop#1320]
```
[2021-07-25 11:46:35,288] INFO in _internal: 192.168.178.54 - - [25/Jul/2021 11:46:35] "ESC[35mESC[1mPOST /auth/login HTTP/1.1ESC[0m" 500 -
[2021-07-25 11:46:35,387] INFO in _internal: 192.168.178.54 - - [25/Jul/2021 11:46:35] "GET /static/styles.css HTTP/1.1" 200 -
[2021-07-25 11:46:38,475] INFO in _internal: 192.168.178.54 - - [25/Jul/2021 11:46:38] "ESC[32mGET / HTTP/1.1ESC[0m" 302 -
[2021-07-25 11:46:38,523] INFO in _internal: 192.168.178.54 - - [25/Jul/2021 11:46:38] "GET /auth/login?next=%2F HTTP/1.1" 200 -
[2021-07-25 11:47:10,205] INFO in _internal: 192.168.178.54 - - [25/Jul/2021 11:47:10] "GET /auth/login?next=%2F HTTP/1.1" 200 -
[2021-07-25 11:47:10,256] INFO in _internal: 192.168.178.54 - - [25/Jul/2021 11:47:10] "GET /static/styles.css HTTP/1.1" 200 -
[2021-07-25 11:47:10,278] INFO in _internal: 192.168.178.54 - - [25/Jul/2021 11:47:10] "GET /static/helpers.js HTTP/1.1" 200 -
[2021-07-25 11:47:20,135] ERROR in controller: Uncaught exception: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
[2021-07-25 11:47:20,136] ERROR in controller: Traceback (most recent call last):
  File "/home/bitcoin/.specter/.env/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/bitcoin/.specter/.env/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/bitcoin/.specter/.env/lib/python3.7/site-packages/cryptoadvance/specter/server_endpoints/auth.py", line 78, in login
    password = request.form["password"]
  File "/home/bitcoin/.specter/.env/lib/python3.7/site-packages/werkzeug/datastructures.py", line 377, in __getitem__
    raise exceptions.BadRequestKeyError(key)
werkzeug.exceptions.BadRequestKeyError: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.

[2021-07-25 11:47:20,144] INFO in _internal: 192.168.178.54 - - [25/Jul/2021 11:47:20] "ESC[35mESC[1mPOST /auth/login HTTP/1.1ESC[0m" 500 -
[2021-07-25 11:47:20,215] INFO in _internal: 192.168.178.54 - - [25/Jul/2021 11:47:20] "GET /static/styles.css HTTP/1.1" 200 -

```

Copied a working and a non-working request from the dev-tools. Here is a working request from a firefox on ubuntu:
```
curl 'https://192.168.178.61:25441/auth/login' 
-H 'User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:90.0) Gecko/20100101 Firefox/90.0' 
-H 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8' 
-H 'Accept-Language: en-US,en;q=0.5' 
--compressed 
-H 'Content-Type: application/x-www-form-urlencoded' 
-H 'Origin: https://192.168.178.61:25441' 
-H 'Connection: keep-alive'
-H 'Referer: https://192.168.178.61:25441/auth/login' 
-H 'Cookie: theme=dark; session=eyJjc3JmX3Rva2VuIjoiMTYyMWJlZGVjNjA4MzNhNmUyOWRmN2M5MWQ2ZGZiNmFiMjU5NzUxNSJ9.YP1FWQ.3iorVk4dbKdusK1VJzxmvYYiF98' 
-H 'Upgrade-Insecure-Requests: 1' 
-H 'Sec-Fetch-Dest: document' 
-H 'Sec-Fetch-Mode: navigate' 
-H 'Sec-Fetch-Site: same-origin'
-H 'Sec-Fetch-User: ?1' 
-H 'Pragma: no-cache' 
-H 'Cache-Control: no-cache' 
--data-raw 'username=test&password=test&next=None&csrf_token=IjE2MjFiZWRlYzYwODMzYTZlMjlkZjdjOTFkNmRmYjZhYjI1OTc1MTUi.YP1FWQ.T8TJEWH72088fRKhpST3kOhHgpw'
```
non-working firefox on windows:
```
curl 'https://192.168.178.61:25441/auth/login' 
-H 'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:90.0) Gecko/20100101 Firefox/90.0' 
-H 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,/;q=0.8' 
-H 'Accept-Language: de,en-US;q=0.7,en;q=0.3' 
--compressed 
-H 'Content-Type: application/x-www-form-urlencoded' 
-H 'Origin: https://192.168.178.61:25441' 
-H 'Connection: keep-alive' 
-H 'Referer: https://192.168.178.61:25441/auth/login?next=%2F' 
-H 'Cookie: session=eyJjc3JmX3Rva2VuIjoiMmY2ZTM4ZTU3ZDE1NWIwODY1ZTIwMmNmZWMxMWMzZjEwMjM4OGYxZCJ9.YP1IwQ.uTWtkOt01CWHx9rLrpNVQX5SvZ8' -H 'Upgrade-Insecure-Requests: 1' 
-H 'Sec-Fetch-Dest: document' 
-H 'Sec-Fetch-Mode: navigate' 
-H 'Sec-Fetch-Site: same-origin' 
-H 'Sec-Fetch-User: ?1' 
--data-raw 'username=test&Passwort=test&next=%2F&csrf_token=IjJmNmUzOGU1N2QxNTViMDg2NWUyMDJjZmVjMTFjM2YxMDIzODhmMWQi.YP1IwQ.CjU2N5Pb0B7gH_PUKWrtAFTyTE4'
```

**Comment by k9ert** [comment]
As we can see, it's the id of the password-field which gets translated from "password" to "Passwort".
Easy fix ...

**Comment by k9ert** [comment]
Fixed with `v1.5.1`
